packages/ui/core-components/src/lib/unsorted/viz/map/components/Points.svelte

```diff
@@ -210,6 +213,64 @@
 
 	let values, minValue, maxValue, colorScale, sizeExtents, maxData, maxSizeSq;
 
+	/** @type {[string]} */
```

Comment:
```suggestion
	/** @type {string[]} */
```
this is the type of a single string tuple